javascript/ql/lib/change-notes/2025-03-10-js-refactor-markdown-table.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Improved the modeling of the `markdown-table` package by replacing the QL implementation with a data model.
```

Comment:
```suggestion
* Improved the modeling of the `markdown-table` package to ensure it handles nested arrays properly.
```
It's best to focus on user-facing changes in these change notes and leave out internal details unless they are relevant for query writers.